docker/docker-compose.objects-apis.yml

```diff
@@ -7,6 +7,7 @@ services:
       - POSTGRES_HOST_AUTH_METHOD=trust
     volumes:
       - ./objects-apis/docker-init-objecttypes-api-db.sql:/docker-entrypoint-initdb.d/init_db.sql
+      - ./objects-apis/docker-init-objects-api-db.sql:/docker-entrypoint-initdb.d/init_db.sql
```

Comment:
this overwrites the previous entry. Either you mount the directory itself and put the SQL scripts in there like `001_objecttypes.sql` and `002_objects.sql`, or just add the extra SQL to the existing file. It's okay to treat objects and objecttypes API as one "unit" here.